charts/csi-vxflexos/values.yaml

```diff
@@ -5,12 +5,26 @@
 # Not recommend to change
 version: v2.8.0
 
+# Defines the image locations of every containers used by the CSI Driver & CSM
+# You can replace them with an internal registry location if needed
 images:
   # "driver" defines the container image, used for the driver container.
-  driverRepository: dellemc
-
+  driver: dellemc/csi-vxflexos:v2.8.0
   # "powerflexSdc" defines the SDC image for init container.
   powerflexSdc: dellemc/sdc:3.6.1
+  # CSI sidecars
+  attacher: registry.k8s.io/sig-storage/csi-attacher:v4.3.0
+  provisioner: registry.k8s.io/sig-storage/csi-provisioner:v3.5.0
+  snapshotter: registry.k8s.io/sig-storage/csi-snapshotter:v6.2.2
+  resizer: registry.k8s.io/sig-storage/csi-resizer:v1.8.0
+  registrar: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.8.0
+  healthmonitor: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.9.0
+
+  # CSM sidecars
+  replication: dellemc/dell-csi-replicator:v1.5.0
+  vgsnapshotter: dellemc/csi-volumegroup-snapshotter:v1.2.0
+  podmon: dellemc/podmon:v1.6.0
+  authorization: dellemc/csm-authorization-sidecar:v1.7.0
 
```

Comment:
These changes needs to be updated in https://github.com/dell/helm-charts/blob/release-v1.8.0/charts/container-storage-modules/values.yaml if applicable